main/java/com/bchupika/user/exception/PhoneNumberException.java

```diff
@@ -0,0 +1,7 @@
+package com.bchupika.user.exception;
+
+public class PhoneNumberException extends RuntimeException {
```

Comment:
```suggestion
public class PhoneNumberValidationException extends RuntimeException {
```

Or you can use one common `ValidationException` instead of creating PhoneNumberValidationException, PasswordNotEqualException, EmailValidationException, etc...